test/tools/test_all_collections.py

```diff
@@ -0,0 +1,44 @@
+'''
+Tests if all collection header files are included in the cxx file.
+'''
+
+import sys
+import argparse
+import yaml
+
+
+def test(yamlfile_path, cxxfile_path):
+    '''
+    Test itself.
+    Takes two parameters, Podio YAML file location and cxx file to be checked.
+    '''
+
+    with open(yamlfile_path, mode='r', encoding="utf-8") as yamlfile:
+        datamodel = yaml.safe_load(yamlfile)
+
+    with open(cxxfile_path, mode='r', encoding="utf-8") as cxxfile:
+        cxxfile_lines = cxxfile.readlines()
+
+    datatypes = datamodel['datatypes']
+
+    for collname in datatypes:
+        include_string = '#include "' + collname.replace('::', '/') + \
+                         'Collection.h"'
+        include_found = False
+        for line in cxxfile_lines:
+            if include_string in line:
+                include_found = True
+
+        if not include_found:
+            print('ERROR: Following collection not included in the cxx file.')
+            print('       ' + collname)
+            sys.exit(2)
```

Comment:
With the set of datatypes that are converted from above, it should then be as simple as doing some `datatypes.issubset(converted_types)` to figure out if there are missing conversions. And then use `datatypes - converted_types` to get a list of all the missing ones.